src/Client.js

```
@@ -44,4 +44,12 @@ export default class Client {
 
     return response.data;
   }
+
+  async forgotPassword(email) {
+    const { protocol, hostname, port } = this.options;
+    const uri = this.buildUri(protocol, hostname, port, '/forgot');
+    const response = await axios.post(uri, { email });
+
+    return response.data;
```

Instead of returning the response, what about don't return anything and throw an error on failure?

---

Done